<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Config</class>
    <widget class="QDialog" name="configWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>480</width>
                <height>270</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Configuration</string>
        </property>
        <layout class="QGridLayout" name="gridLayout" rowstretch="1,0">
            <item row="0" column="0">
                <widget class="QTabWidget" name="tabWidget">
                    <property name="currentIndex">
                        <number>0</number>
                    </property>
                    <widget class="QWidget" name="audioInputTab">
                        <attribute name="title">
                            <string>Audio Input</string>
                        </attribute>
                        <layout class="QGridLayout" name="audioInputGridLayout" rowstretch="0,0,0,0,1"
                                columnstretch="0,1,0,1">
                            <item row="0" column="0">
                                <widget class="QLabel" name="audioInputDeviceLabel">
                                    <property name="text">
                                        <string>Device</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="0" column="1" colspan="3">
                                <widget class="QComboBox" name="audioInputDeviceComboBox"/>
                            </item>
                            <item row="1" column="0">
                                <widget class="QLabel" name="audioInputChannelsLabel">
                                    <property name="text">
                                        <string>Channels</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="1">
                                <widget class="QComboBox" name="audioInputChannelsComboBox"/>
                            </item>
                            <item row="1" column="2">
                                <widget class="QLabel" name="audioInputSampleTypeLabel">
                                    <property name="text">
                                        <string>Sample type</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="3">
                                <widget class="QComboBox" name="audioInputSampleTypeComboBox"/>
                            </item>
                            <item row="2" column="0">
                                <widget class="QLabel" name="audioInputSampleRateLabel">
                                    <property name="text">
                                        <string>Sample rate</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="1">
                                <widget class="QComboBox" name="audioInputSampleRateComboBox"/>
                            </item>
                            <item row="2" column="2">
                                <widget class="QLabel" name="audioInputSampleSizeLabel">
                                    <property name="text">
                                        <string>Sample size</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="3">
                                <widget class="QComboBox" name="audioInputSampleSizeComboBox"/>
                            </item>
                            <item row="3" column="0">
                                <widget class="QLabel" name="audioInputEndianLabel">
                                    <property name="text">
                                        <string>Endianess</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="3" column="1">
                                <widget class="QComboBox" name="audioInputEndianComboBox"/>
                            </item>
                            <item row="3" column="2">
                                <widget class="QLabel" name="audioInputCodecLabel">
                                    <property name="text">
                                        <string>Codec</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="3" column="3">
                                <widget class="QComboBox" name="audioInputCodecComboBox"/>
                            </item>
                            <item row="4" column="0" colspan="4">
                                <spacer name="audioInputVerticalSpacer">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>40</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                        </layout>
                    </widget>
                    <widget class="QWidget" name="audioOutputTab">
                        <attribute name="title">
                            <string>Audio Output</string>
                        </attribute>
                        <layout class="QGridLayout" name="audioOutputGridLayout" rowstretch="0,0,0,0,1"
                                columnstretch="0,1,0,1">
                            <item row="0" column="0">
                                <widget class="QLabel" name="audioOutputDeviceLabel">
                                    <property name="text">
                                        <string>Device</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="0" column="1" colspan="3">
                                <widget class="QComboBox" name="audioOutputDeviceComboBox"/>
                            </item>
                            <item row="1" column="0">
                                <widget class="QLabel" name="audioOutputChannelsLabel">
                                    <property name="text">
                                        <string>Channels</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="1">
                                <widget class="QComboBox" name="audioOutputChannelsComboBox"/>
                            </item>
                            <item row="1" column="2">
                                <widget class="QLabel" name="audioOutputSampleTypeLabel">
                                    <property name="text">
                                        <string>Sample type</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="3">
                                <widget class="QComboBox" name="audioOutputSampleTypeComboBox"/>
                            </item>
                            <item row="2" column="0">
                                <widget class="QLabel" name="audioOutputSampleRateLabel">
                                    <property name="text">
                                        <string>Sample rate</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="1">
                                <widget class="QComboBox" name="audioOutputSampleRateComboBox"/>
                            </item>
                            <item row="2" column="2">
                                <widget class="QLabel" name="audioOutputSampleSizeLabel">
                                    <property name="text">
                                        <string>Sample size</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="3">
                                <widget class="QComboBox" name="audioOutputSampleSizeComboBox"/>
                            </item>
                            <item row="3" column="0">
                                <widget class="QLabel" name="audioOutputEndianLabel">
                                    <property name="text">
                                        <string>Endianess</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="3" column="1">
                                <widget class="QComboBox" name="audioOutputEndianComboBox"/>
                            </item>
                            <item row="3" column="2">
                                <widget class="QLabel" name="audioOutputCodecLabel">
                                    <property name="text">
                                        <string>Codec</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="3" column="3">
                                <widget class="QComboBox" name="audioOutputCodecComboBox"/>
                            </item>
                            <item row="4" column="0" colspan="4">
                                <spacer name="audioOutputVerticalSpacer">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>40</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                        </layout>
                    </widget>
                    <widget class="QWidget" name="dspSettingsTab">
                        <attribute name="title">
                            <string>DSP settings</string>
                        </attribute>
                        <layout class="QGridLayout" name="dspSettingsGridLayout" rowstretch="0,0,1" columnstretch="0,1">
                            <item row="0" column="0">
                                <widget class="QLabel" name="dspSettingsAudioChunkSizeLabel">
                                    <property name="text">
                                        <string>Audio Chunk Size</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="0" column="1">
                                <widget class="QComboBox" name="dspSettingsAudioChunkSizeComboBox"/>
                            </item>
                            <item row="1" column="0">
                                <widget class="QLabel" name="dspSettingsFFTSampleSizeLabel">
                                    <property name="text">
                                        <string>FFT Sample Size</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="1">
                                <widget class="QComboBox" name="dspSettingsFFTSampleSizeComboBox"/>
                            </item>
                            <item row="2" column="0" colspan="2">
                                <spacer name="generalVerticalSpacer">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>40</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                        </layout>
                    </widget>
                </widget>
            </item>
            <item row="1" column="0">
                <widget class="QDialogButtonBox" name="buttonBox">
                    <property name="standardButtons">
                        <set>
                            QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Ok|QDialogButtonBox::Reset
                        </set>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
